napari_plugin_manager/_tests/test_qt_plugin_dialog.py

```diff
@@ -10,16 +11,18 @@
 from napari.plugins._tests.test_npe2 import mock_pm  # noqa
 from napari.utils.translations import trans
 
-from napari_plugin_manager import qt_plugin_dialog
-from napari_plugin_manager.qt_package_installer import InstallerActions
-
-if qtpy.API_NAME == 'PySide2' and sys.version_info[:2] == (3, 11):
+if (qtpy.API_NAME == 'PySide2' and platform.system() != "Linux") or (
```

Comment:
The same error was being seen across the board

Soma variation of òbject cannot be interpreted as an integer`

Following napari I am only testing pyside2 combination for linux with python 3.9 and 3.10